x-pack/plugins/enterprise_search/public/applications/app_search/components/api_logs/api_logs_logic.test.ts

```diff
@@ -213,7 +211,12 @@ describe('ApiLogsLogic', () => {
           ApiLogsLogic.actions.fetchApiLogs({ isPoll: true });
           await nextTick();
 
-          expect(setErrorMessage).toHaveBeenCalledWith(POLLING_ERROR_MESSAGE);
+          expect(flashErrorToast).toHaveBeenCalledWith({
+            id: 'apiLogsPollError',
+            title: 'Could not refresh API log data',
+            text: expect.stringContaining('Please check your connection'),
```

Comment:
Any reason not to use the constants here?

Reply:
Primarily for my own readability when parsing unit tests. It's personally an extra pain for me to check vars or hold vars in my head when I'm scanning unit tests, and breaks me out of my flow of reading tests as if I'm going through the actual view / user actions. I like having a sneak preview of the content without having to open a separate file or view the error in-browser.

Reply:
Constance has brought me over to preferring as well 😄